lib/tasks/migrate_data.rake

```diff
@@ -0,0 +1,10 @@
+namespace :migrate_data do
+  desc 'Migrate old records to have author'
+  task add_author: :environment do
+    dummy_author = User.find_or_create_by(email: 'default_author@simple.org')
+
+    [Facility, Patient, Visit].each do |model|
+      model.where(author: nil).update_all(author_id: dummy_author.id)
```

Comment:
We should use update attribute here. Other
```suggestion
      model.where(author: nil).update_attribute(author_id: dummy_author.id)
```

Reply:
If we use update_all, the updated_at time will change, marking these records as updated in the UI, instead of synced/errored.

Reply:
My bad, updated_all is fine.